src/PeakFinder.hpp

```diff
@@ -0,0 +1,86 @@
+#ifndef SHASTA_PEAKFINDER_HPP
+#define SHASTA_PEAKFINDER_HPP
+
+
+#include <experimental/filesystem>
+#include <stdexcept>
+#include <iostream>
+#include <utility>
+#include <vector>
+
+using std::experimental::filesystem::create_directories;
+using std::experimental::filesystem::path;
+using std::runtime_error;
+using std::ostream;
+using std::pair;
+using std::vector;
+using std::string;
+using std::cout;
+
+namespace shasta {
+    class Peak;
+    class PeakFinder;
+}
+
+class shasta::Peak {
+public:
+    /// Attributes ///
```

Comment:
Perhaps make most of this `private`?

Reply:
Or, perhaps better, make `Peak` a private subclass of `PeakFinder`, since it does not to be used outside not it?

Reply:
I could imagine a use case outside of shasta where you would want to do things with a sorted list of Peak objects, but I agree in this context it doesn't apply